octobot_trading/exchanges/util/exchange_market_status_fixer.pxd

```diff
@@ -16,25 +16,24 @@
 #  License along with this library.
 
 cpdef bint is_ms_valid(object value, bint zero_valid=*)
-
-cdef void calculate_costs(dict market_limit)
-cdef void calculate_prices(dict market_limit)
-cdef void calculate_amounts(dict market_limit)
-cdef object get_markets_limit(dict market_limit)
-cdef void fix_market_status_limits_from_current_data(dict market_limit)
-cdef bint check_market_status_limits(market_limit)
-cdef bint check_market_status_values(market_limit, bint zero_valid=*)
+cpdef bint check_market_status_limits(market_limit)
```

Comment:
Why do we need this functions to be public ?

Reply:
every commit after tmp is a test, I'm trying to figure out why the compiled version is segfaulting

Reply:
it looks like when making these private, python is enable to import them from the ExchangeMarketStatusFixer class

Reply:
:open_mouth:

Reply:
further tests in progess

Reply:
Issue fixed, a function was imported in another module